lib/sanbase/clickhouse/historical_balance/fetchers/utxo_sql_queries.ex

```diff
@@ -126,4 +126,23 @@ defmodule Sanbase.Clickhouse.HistoricalBalance.UtxoSqlQueries do
     args = [interval, span, addresses, from_unix, to_unix]
     {query, args}
   end
+
+  def btc_top_transactions_query(from, to, limit) do
+    to_unix = DateTime.to_unix(to)
+    from_unix = DateTime.to_unix(from)
+
+    # only > 500 BTC transfers if range is > 1 week, otherwise only bigger than 100
+    amount_filter = if Timex.diff(to, from, :days) > 7, do: 500, else: 100
+
+    query = """
+    SELECT toUnixTimestamp(dt), address, balance - oldBalance as amount, txID
+    FROM btc_balances
```

Comment:
```suggestion
    FROM btc_balances FINAL
```
to avoid duplicates :?